server/controllers/finance/reports/cashflow/index.js

```diff
@@ -385,7 +387,7 @@ function report(req, res, next) {
     });
     return totals;
   }
-
+  // eslint-disable-next-line no-shadow
```

Comment:
Same remark as for `aggregateTotalByTextKeys`, try to reuse it by removing this definition of the function.